openstack/octavia/templates/etc/_secrets.conf.tpl

```diff
@@ -11,7 +11,7 @@ username = {{ .Release.Name }}
 password = {{ .Values.global.octavia_service_password | replace "$" "" }}
 
 [database]
-connection = {{ include "db_url_mysql" . }}
+connection = {{ include "utils.db_url" . }}
```

Comment:
The [definition](https://github.com/sapcc/helm-charts/blob/master/openstack/utils/templates/_hosts.tpl#L59-L67) reads to me like this config would default to `utils._db_url_mariadb`, because `dbType` isn't set, right? Not sure whether that's what you want.

Reply:
This is the expected behavior. If `dbType` isn't set, then the this value would be rendered the same as if `dbType=mariadb`, eg current behavior.